### DISCUSSION TITLE
How is crisp intergrated into the Retype site

### DISCUSSION THREAD

**Discussion by robwilde** [retypeapp/retype#264] [Q&A] [answered]
I must say how much I love this system, so easy but so beautiful and functional.

I would like to be able to allow people to contact me if needing more info and I like how the crisp chat box is added as an integration on  retype.com website.

I have signed up with crisp and added the integration to the retype.yml but does not work. Is this something easy to implement?

**Comment by fabriciomurta** [comment, marked as answer]
We don't use any special retype version to build retype website at all...

https://github.com/retypeapp/retype/blob/main/retype.yml#L47-L48

This should be all you need to set up the integration (besides getting the key from crisp itself).

**Reply by robwilde** [reply]
Thanks for confirming that. I realised I messed up and didn't spell integrations correctly. It is now working 
thanks
